salt/journal/config/etc-nginx-traits.d-strict-transport-security.conf

```diff
@@ -0,0 +1,9 @@
+if ($https) {
+    add_header Strict-Transport-Security "max-age={{ 60 * 60 * 24 * 365 }}; includeSubDomains";
```

Comment:
`includeSubDomains` can be problematic - may break things like `api.elifesciences.org`, `some-crm.elifesciences.org`, `new-service-we-don-t-know-about.elifesciences.org`... that are not related to journal